app/utils/mail_carrier.rb

```diff
@@ -0,0 +1,15 @@
+module MailCarrier
+
+  module_function
+
+  # This is just a placeholder. Delivering later
+  # hasn't been implemented you.
+  def deliver_later(message)
+    deliver_now(message)
+  end
+
+  def deliver_now(message)
+    message.deliver_now
+  end
```

Comment:
Does this endpoint need to make a separation between deliver_now and deliver_later? What if we just have deliver and later we make it so that the only option is to deliver_later, because that's really what we should always do (deliver from worker).